Description: Configure transpilation as per different versions and transpilers
Author: Bastien Roucariès
Forwarded: not-needed
Reviewed-By: Xavier Guimard <yadd@debian.org>
Last-Update: 2020-09-16

--- a/legacy/rollup.config.js
+++ b/legacy/rollup.config.js
@@ -1,4 +1,22 @@
-import buble from 'rollup-plugin-buble';
+var plugins = [];
+
+switch(process.env.TRANSPILE) {
+case 'babel7':
+    var babel=require('rollup-plugin-babel');
+    const plugins_babel7 = [
+      babel(),
+    ];
+    plugins = plugins_babel7; break;
+case 'none':
+    plugins = []; break;
+default:
+    var buble=require('rollup-plugin-buble');
+    const plugins_buble = [
+    buble()
+    ];
+    plugins = plugins_buble;
+}
+
 import pkg from './package.json';
 
 export default {
@@ -12,9 +30,7 @@
 		'typescript'
 	],
 
-	plugins: [
-		buble()
-	],
+	plugins: plugins,
 
 	output: [
 		{
