Description: fix for current tsc & rollup 2
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2020-10-04

--- a/packages/typescript/src/index.ts
+++ b/packages/typescript/src/index.ts
@@ -160,7 +160,7 @@
 
       const output = findTypescriptOutput(ts, parsedOptions, id, emittedFiles, tsCache);
 
-      return output.code != null ? (output as SourceDescription) : null;
+      return output["code"] != null ? (output as SourceDescription) : null;
     },
 
     async generateBundle(outputOptions) {
--- a/packages/typescript/src/outputFile.ts
+++ b/packages/typescript/src/outputFile.ts
@@ -10,6 +10,8 @@
 import type TSCache from './tscache';
 
 export interface TypescriptSourceDescription extends Partial<SourceDescription> {
+  code: string;
+  map: string;
   declarations: string[];
 }
 
